cloud/services/compute/loadbalancers.go

```diff
@@ -198,10 +200,12 @@ func (s *Service) DeleteLoadbalancers() error {
 		name := s.getAPIServerIPAddressSpec().Name
 		op, err := s.addresses.Delete(s.scope.Project(), name).Do()
 		if err != nil {
-			return errors.Wrapf(err, "failed to delete forwarding rules")
+			if !gcperrors.IsNotFound(err) {
```

Comment:
Similar to above, this conditional can be combined with the `err != nil` check above